lms/djangoapps/instructor/tests/test_certificates.py

```diff
@@ -136,9 +136,9 @@ def _assert_certificates_visible(self, is_visible):
         """Check that the certificates section is visible on the instructor dash. """
         response = self.client.get(self.url)
         if is_visible:
-            self.assertContains(response, "Certificates")
+            self.assertContains(response, "Student-Generated Certificates")
```

Comment:
Actually, a new section `certificate` has been added in instructor dashbord and this test is comparing the `certificate` string in the response while `data download` section has the certificate strings as well (reflect of this PR).